doc/md/getting-started.md

```diff
@@ -84,9 +84,11 @@ import (
 func (User) Fields() []ent.Field {
 	return []ent.Field{
 		field.Int("age").
-			Positive(),
+			Positive().
+			Comment("Age of the user in years."),
 		field.String("name").
-			Default("unknown"),
+			Default("unknown").
+			Comment("Name (first last) of the user. Defaults to \"unkown\"."),
```

Comment:
You can add it to the fields/edges pages, no need for this in the getting-started page.